.github/scripts/update-release-notes.js

```diff
@@ -0,0 +1,160 @@
+/*
+Copyright 2023 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+var https = require('https');
+const fs = require("fs");
+const execSync = require('child_process').execSync;
+
+const token = process.argv[2];
+
+if(token == undefined) {
+    throw new Error("token is undefined")
+}
+
+function isEarlierThan24Hours(timestampInMilliseconds) {
+    const timestamp = new Date(timestampInMilliseconds);
+    const now = new Date();
+    return timestamp < now - (24 * 60 * 60 * 1000);
+  }
+
+function fetchReleaseNotes(owner,repo, tag, callback) {
+    var optionsget = {
+        host : 'api.github.com', 
+        port : 443,
+        path :  `/repos/${owner}/${repo}/releases/tags/${tag}`, 
+        method : 'GET',
+        headers: {
+            'Accept': 'application/vnd.github+json',
+            'User-Agent': 'server-side',
+            'X-GitHub-Api-Version': '2022-11-28',
+            'Authorization': `Bearer ${token}`,
+          }
+    };
+    var reqGet = https.request(optionsget, function(res) {
+        // console.log("statusCode: ", res.statusCode);
+        var data = [];
+        res.on('data', function(chunk) {
+            data.push(chunk);
+        }).on('end', function() {
+            var buffer = Buffer.concat(data);
+            str = new TextDecoder("utf-8").decode(buffer)
+            responseJson = JSON.parse(str)
+            
+            callback(responseJson.published_at, responseJson.body)
+            
+        });
+     
+    });
+     
+    reqGet.end();
+    reqGet.on('error', function(e) {
+        console.error(e);
+    });
+}
+
+
+function updateBOMReleaseNotes() {
+var optionsget = {
+    host : 'search.maven.org', 
+    port : 443,
+    path :  '/solrsearch/select?q=g:com.adobe.marketing.mobile+AND+a:sdk-bom&core=gav&rows=2&wt=json', 
+    method : 'GET',
+    headers: {
+        'Content-Type': 'application/json',
+        'User-Agent': 'server-side',
+      }
+};
+
+var reqGet = https.request(optionsget, function(res) {
+    // console.log("statusCode: ", res.statusCode);
+ 
+    res.on('data', function(d) {
+        str = new TextDecoder("utf-8").decode(d)
+        responseJson = JSON.parse(str)
+
+        responseJson.response.docs.forEach(element => {
+            if (isEarlierThan24Hours(element.timestamp) == false) {
+                if (element.a == 'sdk-bom') {
```

Comment:
nit: Can we combine the two checks in one instead?

Reply:
I will refactor this function to work with other Android extensions in another PR. I'd suggest keeping this for now.